content/zh/docs/toolbox/auditing/auditing-receive-customize.md

```diff
@@ -101,19 +101,19 @@ preferences: {}
 users: []
 ```
 
- The `ip` is the `CLUSTER-IP` of Service `kube-auditing-webhook-svc` in the namespace `kubesphere-logging-system`. You can get it using this command.
+`ip` 即命名空间 `kubesphere-logging-system` 中 `kube-auditing-webhook-svc` 服务的 `CLUSTER-IP`，您可以使用以下命令来获取。
 
 ```bash
 kubectl get svc -n kubesphere-logging-system
 ```
 
 {{< notice note >}}
 
-It should restart the Kubernetes apiserver to make the changes effective after you modified these two files.
+修改这两个文件后，应重启 Kubernetes Apiserver 使修改生效。
```

Comment:
Change the English source text to:

You need to restart the Kubernetes apiserver to make the changes effective after you modified these two files.